site/en/install/lang_java.md

```diff
@@ -24,61 +24,57 @@ TensorFlow Java runs on Java 8 and above, and supports out-of-the-box the follow
 ## Versions
 
 TensorFlow Java has its own release cycle, independent from the [TensorFlow runtime](https://github.com/tensorflow/tensorflow).
-In consequence, its version does not match the version of TensorFlow runtime it runs on. Consult the TensorFlow Java 
+Consequently, its version does not match the version of TensorFlow runtime it runs on. Consult the TensorFlow Java 
 [versioning table](https://github.com/tensorflow/java/#tensorflow-version-support) to list all versions available 
 and their mapping with the TensorFlow runtime.
 
 
-## Installing with Maven
+## Artifacts
 
-To include TensorFlow in your [Maven](http://maven.apache.org) application, you first need to add in
-your project's `pom.xml` file a dependency on one of the `tensorflow-core-platform` artifact.
+There are [several ways](https://github.com/tensorflow/java/#using-maven-artifacts) to add TensorFlow Java to your project. 
+The easiest one is to add a dependency to the `tensorflow-core-platform` artifact, which not only include the
```

Comment:
"dependency on the", "which includes both the TensorFlow Java Core API and the ..."